### ISSUE TITLE
Moon support for Yeelight ceiling lights

### ISSUE THREAD

**Issue by S474N** [mattmattmatt/node-red-contrib-yeelight-compat-hue#24]
There is property `active_mode` with values` 0: daylight mode / 1: moonlight mode (ceiling light only)`

Is it possible to add this feature?

Is also in Yeelight API on page 21:
https://www.yeelight.com/download/Yeelight_Inter-Operation_Spec.pdf

**Comment by asm0dey** [comment]
Library 'yeelight2' have also supported it

**Comment by S474N** [comment]
@asm0dey what do you mean exactly please? Another plugin?

**Comment by asm0dey** [comment]
I mean this plugin depends on yeelight2 library (https://github.com/mattmattmatt/node-red-contrib-yeelight-compat-hue/blob/master/package.json#L101), which already supports this functionality:

**Comment by S474N** [comment]
@asm0dey and how can I used it?

**Comment by asm0dey** [comment]
I believe @mattmattmatt can :)

**Comment by S474N** [comment]
@asm0dey  yes, I hope so :D

**Comment by S474N** [comment]
@asm0dey do you know about another plugin, which works with moon support?

**Comment by asm0dey** [comment]
@S474N nop. And you?

**Comment by S474N** [comment]
Nope. I'm using also next instance of Homebridge with Yeelighter, but there are problems with it. :(

**Comment by jpg06** [comment]
The development of the plugin seems to be stopped and "mattmattmatt" does not respond to "pullrequest".
If I have some time I'll see if I can make the modification in the version I forked.
This possibility interests me too ;)

**Comment by asm0dey** [comment]
@jpg06 that's very interesting! Can't wait

**Comment by S474N** [comment]
@jpg06 go go go :)

**Comment by jpg06** [comment]
I made a first functional version (local for now).
But there is a bug with the latest version of Yeelight2 (1.3.7)
I think I have found a way to resolve it and I will propose the fix to the author of the module.
It is also necessary to implement a missing parameter in a function of the Yeelight2 module: I made a pullrequest for it.
See you soon.

**Comment by S474N** [comment]
Excellent! Will you fork this plugin?

**Comment by S474N** [comment]
Can you send your local version for testing please?

**Comment by S474N** [comment]
Problem with 1.3.7 is solved, right? Progress with your fork? Thanks for reply.

**Comment by asm0dey** [comment]
@jpg06 ?

**Comment by S474N** [comment]
@jpg06 some progress?

BTW do somebody knows telnet command to Moon mode?

**Comment by S474N** [comment]
@jpg06 progress?

**Comment by GinBook** [comment]
Is there any progress? The MOON feature is really missing from the settings and it's sad that there is no update to this plug-in all the time.
